src/modules/Snapper.rb

```diff
@@ -27,6 +27,7 @@
 # Input and output routines.
 
 require "yast"
+require_relative '../lib/snapper/snapshot'
```

Comment:
no, correct usage is `require "../lib/snapper/snapshot"` also please respect coding style and use double quotes. BTW some modules already use rubocop that warn you, but not this one.